core/src/main/java/com/digitalpebble/stormcrawler/persistence/urlbuffer/URLBuffer.java

```diff
@@ -85,29 +89,4 @@ default void acked(String url) {
     }
 
     default void configure(Map<String, Object> stormConf) {}
-
-    /** Returns a URLBuffer instance based on the configuration * */
-    public static URLBuffer getInstance(Map stormConf) {
```

Comment:
This might break end user code (removal of a public method), so we need to be clear in the changelog.